libwayshot/src/image_util.rs

```diff
@@ -35,14 +38,22 @@ pub(crate) fn rotate_image_buffer(
         _ => image,
     };
 
-    if final_image.dimensions() == (width, height) {
-        return final_image;
+    let scale = rotated_image.width() as f64 / logical_width as f64;
```

Comment:
emm.. no , I do not think every picture should resize. 

I think if the dimesions is the same. it should not resize. the final image to file should be clear enough, so I think resize should done in freeze image, not all images

Reply:
It's not doing any resizing if it's not needed, see line 44.

Reply:
This logic looks correct to me. Resolving.